plugins/module_utils/utils.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
do we want to start to write unit tests for all these newly introduced utils function?